src/main/java/uk/gov/hmcts/reform/sscs/service/SubmitAppealService.java

```diff
@@ -111,42 +109,12 @@ private SscsCaseDetails createCaseInCcd(SscsCaseData caseData, IdamTokens idamTo
         }
     }
 
-    protected SscsCaseData transformAppealToCaseData(SyaCaseWrapper appeal) {
-        SscsCaseData caseData = submitYourAppealToCcdCaseDataDeserializer.convertSyaToCcdCaseData(appeal);
-
-        return updateCaseData(caseData);
-    }
-
-    protected SscsCaseData transformAppealToCaseData(SyaCaseWrapper appeal, String region, RegionalProcessingCenter rpc) {
-
-        SscsCaseData caseData = submitYourAppealToCcdCaseDataDeserializer.convertSyaToCcdCaseData(appeal, region, rpc);
-
-        return updateCaseData(caseData);
+    private SscsCaseData transformAppealToCaseData(SyaCaseWrapper appeal) {
+        return submitYourAppealToCcdCaseDataDeserializer.convertSyaToCcdCaseData(appeal, appealNumberGenerator);
```

Comment:
why do you need to pass the appealNumberGenerator as a parameter? it's not used anywhere else in this class, right? 
I would actually make the AppealNumberGenerator.generateAppealNumber() method static.